docs/format/db.rst

```diff
@@ -58,6 +71,29 @@ Related tools:
    * Ex: CLB is solved by first solving LUT bits, and then solving FF bits
 
 
+Interconnect PIP Tags
+^^^^^^^^^^^^^^^^^^^^^
+
+Tags for interconnect PIPs are stored in the `segbits_int_l.db` and `segbits_int_r.db` database files. For example, look at `segbits_int_l.db` for the bits that configure the `INT_L` tile in a `CLBLL_L` or `CLBLM_L` segment.
```

Comment:
This sentence is weird, INT_L has nothing to do with CLBLM_L, it is the general interconnect switch box.  The pips are located in the tile, therefore you look at that segbit.

Reply:
Deleted.